src/main/java/io/github/oliviercailloux/y2018/apartments/valuefunction/profile/Profile.java

```diff
@@ -18,20 +18,10 @@
   private LinearAVF linearAvf;
 
   /**
-   * The 10 next arguments gives the Range of apartment characteristic subjective value weight in
-   * the calculation of the Apartment total subjective value
+   * rangeMap is a map of criterions containing the ranges of all the different range for these
+   * criterions
    */
-  private Range<Double> floorAreaWeightRange;
-
-  private Range<Double> nbBedroomsWeightRange;
-  private Range<Double> nbSleepingWeightRange;
-  private Range<Double> nbBathroomsWeightRange;
-  private Range<Double> terraceWeightRange;
-  private Range<Double> floorAreaTerraceWeightRange;
-  private Range<Double> wifiSubjectiveValueWeightRange;
-  private Range<Double> pricePerNightSubjectiveValueWeightRange;
-  private Range<Double> nbMinNightSubjectiveValueWeightRange;
-  private Range<Double> teleSubjectiveValueWeightRange;
+  private HashMap<Criterion, Range<Double>> rangeMap;
```

Comment:
Utiliser une EnumMap plutôt qu'une HashMap (j'ai donné un exemple sur la PR #55).

Reply:
Fixed in commit 2d6652c